iGCS/UserInterface/ViewControllers/GCSMapViewController.m

```diff
@@ -437,14 +448,11 @@ - (void) handlePacket:(mavlink_message_t*)msg {
             [self.customModeLabel setText:[MavLinkUtility mavCustomModeToString:  heartbeat]];
 
             NSInteger idx = CONTROL_MODE_RC;
-            switch (heartbeat.custom_mode) {
-                case AUTO:
-                    idx = CONTROL_MODE_AUTO;
-                    break;
-
-                case GUIDED:
-                    idx = CONTROL_MODE_GUIDED;
-                    break;
+
+            if (heartbeat.custom_mode == APMPlaneAuto || heartbeat.custom_mode == APMCopterAuto) {
+                idx = CONTROL_MODE_AUTO;
+            } else if (heartbeat.custom_mode == APMPlaneAuto || heartbeat.custom_mode == APMCopterAuto) {
```

Comment:
Looks like another mode typo here.

Will fix on IGCS-236b branch - just marking this here in case we reject that branch.